lib/components/Editor/CustomExtensions/BubbleMenu/TextOptions.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { getTextMenuOptions } from "./constants";
+import LinkOption from "./LinkOption";
+import Option from "./Option";
+
+const TextOptions = ({
+  editor,
+  options,
+  setIsInvalidLink,
+  isLinkOptionActive,
+  setIsLinkOptionActive,
+}) => {
+  const handleAnimateInvalidLink = () => {
+    setIsInvalidLink(true);
+    const timer = setTimeout(() => {
+      setIsInvalidLink(false);
+      clearTimeout(timer);
```

Comment:
The clearTimeout here may not have any effect as this will be only evaluated after the timeout has already triggered and cleared.